tests/src/test/scala/scalaz/MaybeTests.scala

```diff
@@ -82,6 +82,13 @@ object MaybeTest extends SpecLite {
 
   "empty isn't just" ! check(!empty.isJust)
 
+  "empty pattern matches" ! check {
```

Comment:
we don't really need this, but I just wanted to be sure I didn't break something.